.github/workflows/ci_linux.yml

```diff
@@ -1,47 +1,47 @@
-name: CI (Linux)
-
-on:
-  push:
-    branches: [ main ]
-  pull_request:
-    branches: [ main ]
-
-jobs:
-  build:
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@v3
-
-      - name: Use Node.js 16
-        uses: actions/setup-node@v3
-        with:
-          node-version: 16
-
-      - name: Install Dependencies
-        run: yarn install
-        working-directory: Extension
-
-      - name: Compile Sources
-        run: yarn run compile
-        working-directory: Extension
-
-      - name: Run Linter
-        run: yarn run lint
-        working-directory: Extension
-
-      - name: Compile Test Sources
-        run: yarn run pretest
-        working-directory: Extension
-
-      - name: Run unit tests
-        uses: GabrielBB/xvfb-action@v1.6
-        with:
-          run: yarn run unitTests
-          working-directory: Extension
-
-#      - name: Run languageServer integration tests
-#        uses: GabrielBB/xvfb-action@v1.6
-#        with:
-#          run: yarn run integrationTests
-#          working-directory: Extension
+name: CI (Linux)
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest

```

Comment:
Can this be changed to ubuntu-22 so that potential OS changes don't cause a potential break unexpectedly?

Reply:
I've updated them to use `ubuntu-22.04`, `windows-2022` and `macos-12`.